src/commands/configuration/config-subcommands/shoutout.ts

```diff
@@ -0,0 +1,115 @@
+            o.setName("channel")
+                .setDescription("The channel to use as a shout out channel. Omit to remove configuration")
+                .addChannelTypes(ChannelType.GuildText)
+        )
+        .addRoleOption(o =>
+            o.setName("role")
+                .setDescription("The role to auto shout out. Omit to remove configuration")
+        )
+        .addStringOption(o =>
+            o.setName("message")
+                .setDescription("The message to shout out with. Omit to use the default message")
+        ),
+
+    // Command execution
+    async execute(ctx: ChatInputCommandInteraction): Promise<void> {
+        // Default values for parameters
+
+        const channel: TextChannel = ctx.options.getChannel("channel") ?? null;
+        const role: Role = <Role> ctx.options.getRole("role") ?? null;
+        const message: string = ctx.options.getString("message") ?? null;
+
+        await ctx.deferReply();
+
+        // Get the server document from the database
+        const server: Server = await Server.get(ctx.guildId);
+
+        // Remove configurations if channel or role was omitted
+        if (!channel || !role) {
+            server.shoutout = null;
+            await server.save();
+            await ctx.followUp("Set to not auto shout out");
+            return;
+        }
+        
+        server.shoutout = {
+            channel: channel.id,
+            role: role.id,
+            message: message
+        };
+        await server.save();
+
+        await ctx.followUp(`Set to auto shout out ${role} in ${channel}`);
+    },
+    // Error handler
+    error: defaultErrorHandler,
+
+    // Help command embed
+    help: new EmbedBuilder()
+        .setTitle("Shoutout")
+        .setDescription(
+            "A server configutation command that sets the auto shout out channel, role, and message for a server"
+        )
+        .addFields(
+            { name: "Format", value: `\`/config ${name} [channel] [role] [message]\`` },
+            { name: "[channel]", value: "Optional parameter. The channel to set as auto shout out channel. Omit to remove auto shout outs" },
+            { name: "[role]", value: "Optional parameter. The role to set as auto shout out role. Omit to remove auto shout outs" },
+            {
+                name: "[message]",
+                value: "Optional parameter. The auto shout out message. Omit to use the default message\n" +
+                       "Variables:\n" +
+                       `- \`${DISCORD_NAME}\` - Replaced with Discord display name\n` +
+                       `- \`${CHANNEL}\` - Replaced with Twitch username\n` +
+                       `- \`${LINK}\` - Replaced with Twitch link\n` +
+                       `- \`${TITLE}\` - Replaced with Stream title\n` +
+                       `- \`${GAME}\` - Replaced with Stream game` +
+                       `- \`${NEW_LINE}\` - Replaced with a new line`
+            }
+        )
+};
```

Comment:
This is the `/config shoutout`. It lets server administrators configure auto shout outs by choosing a channel to post shout outs, a role to shout out, and a tessage template to use.